### PR TITLE
Updating README for beta release

### PR DESCRIPTION
**Pull request by NivekT**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #264

Updating README for beta release. If you have time, please read through the whole file and let me know what should change.

@ejguan - anything to change for installation?

Differential Revision: [D34488942](https://our.internmc.facebook.com/intern/diff/D34488942)

### CODE REVIEW

**Comment by NivekT** [comment]
@nivekt has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D34488942).

File: README.md [hunk 1]
```diff
@@ -151,149 +136,73 @@ Under this naming convention, `DataSet` simply refers to a graph of `DataPipes`,
...
-For a complete example, suppose we want to load data from CSV files with the following steps:
+## Contributing
 
-- List all csv files in a directory
-- Load csv files
-- Parse csv file and yield rows
+We welcome PRs! See the [CONTRIBUTING](CONTRIBUTING.md) file.
 
-To support the above pipeline, `CSVParser` is registered as `parse_csv` to consume file streams and expand them as rows.
+## Building the Documentation
```
Comment (ejguan) [inline README.md:181]: Can you move this part into https://github.com/pytorch/data/blob/main/docs/README.md?

Comment (NivekT) [reply README.md]: Done

**Review by ejguan — APPROVED** [review]
LGTM

**Comment by NivekT** [comment]
@nivekt has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D34488942).

**Comment by NivekT** [comment]
@nivekt has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D34488942).

**Pull request closed without merging**